tests/backstop/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"scripts": {
+		"pretest:visualtest": "cd ../../ && VRT=1 npm run build:storybook",
+		"test:visualtest": "npm run backstopjs -- test",
+		"test:visualapprove": "npm run backstopjs -- approve",
+		"backstopjs": "./bin/backstop"
+	},
+	"dependencies": {
```

Comment:
Plus `glob`, `node-fetch`, `@babel/parser` and `@babel/traverse`. The last two don't exist as direct dependencies in the main `package.json` file, but we still need to add it explicitly to the workspaces package.json.